client/src/components/geneExpression/geneSet.js

```diff
@@ -0,0 +1,199 @@
+// jshint esversion: 6
```

Comment:
Unless I'm mistaken, you don't need to add these jshints anymore.  They are ignored.